build/blueprints/template/files/src/patternfly/__typeDirectory__/__namePascalized__/examples/__namePascalized__.md

```diff
@@ -2,14 +2,43 @@
 title: '{{nameCapitalized}}'
 section: beta
 beta: true
-cssPrefix: pf-c-{{nameCapitalized}}
+cssPrefix: pf-c-{{nameDasherized}}
 ---
 
+## Optional introduction
+
+Optional introduction for `{{nameDasherized}}`.
+
 ## Examples
+
 ```hbs title=Basic
 {{moduleHbOpen}}
-  Basic example content
+  {{moduleHbIdConcatenationOpen}}
+    Item
+  {{moduleHbIdConcatenationClose}}
 {{moduleHbClose}}
 ```
 
 ## Documentation
+
+### Overview
+
+`{{nameDasherized}}` is a component. Some of its elements are presentational, while others require accessibility support.
+
+### Accessibility
+
+| Attribute | Applied to | Outcome |
+| -- | -- | -- |
+| `hidden` | `.{{nameDasherized}}__item` |  Indicates that the `.{{nameDasherized}}__item` is hidden. |
+| `aria-hidden` | `.{{nameDasherized}}__item` |  Indicates that the `.{{nameDasherized}}__item` is hidden from and unfocusable to screen readers, but remains visible for sighted users. |
```

Comment:
If the description is that the thing is hidden, we'll need to specify the value as true since the default value is false.

```suggestion
| `aria-hidden="true"` | `.{{nameDasherized}}__item` |  Indicates that the `.{{nameDasherized}}__item` is hidden from and unfocusable to screen readers, but remains visible for sighted users. |
```